netbox/templates/dcim/device.html

```diff
@@ -313,8 +313,8 @@ <h5 class="card-header">Dimensions</h5>
                         <tr>
                             <th scope="row">Weight</th>
                             <td>
-                                {% if object.device_type.weight %}
-                                    {{ object.device_type.weight|floatformat }} {{ object.device_type.get_weight_unit_display }}
+                                {% if object.total_weight %}
+                                    {{ object.total_weight|floatformat }} {{ object.device_type.get_weight_unit_display }}
```

Comment:
I believe the total weight will always be displayed in kilograms.
```suggestion
                                    {{ object.total_weight|floatformat }} Kilograms
```

Reply:
Ok, I wasn't sure of that. I'll make this change now